wordpress-mobile/WordPress-Android
# Use a test host to speed up connected tests

This PR dramatically speeds up the WordPressUtils Connected Tests by implementing an empty test host `apk` rather than compiling all of WordPress Android for use as one.

**To test:**
- Ensure CI passes
- Ensure that the "WordPressUtils Connected Tests" step now runs [faster than 4 minutes](https://app.circleci.com/pipelines/github/wordpress-mobile/WordPress-Android/9953/workflows/9d268469-fa29-4bba-a823-5e18f635a094/jobs/42185).

PR submission checklist:
